deploy/hub/hub_controller_clusterrole.yaml

```diff
@@ -38,3 +38,7 @@ rules:
 - apiGroups: ["register.open-cluster-management.io"]
   resources: ["managedclusters/accept"]
   verbs: ["update"]
+# Allow hub to read cluster version
```

Comment:
can this comment be updated to say why it needs this access?

Reply:
added comments David.